src/scripts/html-transforms.js

```diff
@@ -62,8 +62,14 @@ export function removeStyleAndScript (document) {
 export function addTargetBlank (document) {
   // Add target="_blank" to all <a>tags
   document.querySelectorAll('a').forEach(node => {
-    node.setAttribute('target', '_blank');
+    // set href to empty string in case attribute is null
+    const href = node.getAttribute('href') || '';
+
+    if (href.charAt(0) !== '#' && href.indexOf('javascript:') !== 0) {
```

Comment:
I think our browser requirements mean we can use `String.startsWith()`, which would probably read a little better for both of these checks.